### ISSUE TITLE
DeepLung pretrained weights gets changed while importing weights from Pytorch to ONNX.

### ISSUE THREAD

**Issue by 91parthparikh** [pytorch/pytorch#72392]
### 🐛 Describe the bug

While importing pre-trained weights from Pytorch to ONNX, weights are getting modified. Well I have 1.9 pytorch version and used below script for converting a pre-trained weights from pytorch to ONNX.

Weights can be found from the below path:
https://github.com/wentaozhu/DeepLung/tree/master/detector/dpnmodel
```

import torch
import numpy
import onnx
import dpn3d26 as dp
from scipy.io import savemat

cfg = { 'in_planes': (24,48, 72, 96), 'out_planes': (24,48,72,96), 'num_blocks': (2,2,2,2), 'dense_depth':(8,8,8,8)}

model = dp.DPN(cfg)
state_dict = torch.load('fd9044.ckpt')
model.load_state_dict(state_dict['state_dict'])

model.eval()

x = (torch.randn(1,1,96,96,96,requires_grad=True))
crd = (torch.randn(1,3,24,24,24,requires_grad=True))

torch.onnx.export(model,(x,crd),'newdetection.onnx',export_params=True,
                  opset_version=9,do_constant_folding=True,input_names=['input', 'coord'],
                  output_names=['output'],dynamic_axes={'input' : {0: 'batch_size'},
                                                        'coord' : {0: 'batch_size'},
                                                        'output' : {0: 'batch_size'}})

layerWeights = model.layer1[0].conv1.weight

weights = layerWeights.detach().numpy()
savemat("matlab_matrix_SecondConv.mat", {'weights': weights})
```

### Versions

The issue exists in torch->onnx conversion. I have compared the inference between ONNX and MATLAB, which is working well. Another interesting thing is, pytorch version of the model has Batch Normalization after every conv3d layer (conv3d --> BN --> relu), whereas converted ONNX version has only relu layer (conv3d --> relu).

 Compare above saved weights with imported ONNX weights in MATLAB

```
% Weights comparison comparison
load matlab_matrix_SecondConv.mat

out = importONNXLayers('newdetection.onnx');
matlabWeights = out.Layers(5, 1).Weights;
```

**Comment by BowenBao** [comment]
@91parthparikh Hi, the reason weights being changed is due to that ONNX export applies ConvBN fusion. This only speeds up computation, and does not affect the model final output.

**Comment by 91parthparikh** [comment]
Hi @BowenBao , thanks for looking into this. Initially I had same impression but after investigating more, I found that final output of the model is getting modified (accuracy is getting dropped). Is there any way in ONNX where we can export weights without having fusion ?

**Comment by BowenBao** [comment]
Hi @91parthparikh unfortunately there isn't any api to disable the fusion. Please note the accuracy drop might have a different root cause. There is a few things you can try
1. Try with latest pytorch to see if the accuracy improves.
2. Manually disable fusion by removing line at https://github.com/pytorch/pytorch/blob/84729cef70a7184a06bffb14cff049debe8c9471/torch/onnx/utils.py#L533

**Comment by 91parthparikh** [comment]
Hi @BowenBao , Thanks for the solution. I have opted for 2nd solution. Closing this loop here.
